core/controllers/admin.py

```diff
@@ -205,6 +290,56 @@ def _reload_collection(self, collection_id):
             raise Exception('Cannot reload a collection in production.')
 
 
+class AdminRoleHandler(base.BaseHandler):
+    """Handler for roles tab of admin page. Used to view and update roles."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @require_super_admin
+    def get(self):
+        view_method = self.request.params['method']
+
+        if view_method == feconf.VIEW_METHOD_ROLE:
+            role = self.request.params[feconf.VIEW_METHOD_ROLE]
```

Comment:
Btw this seems odd. Don't we normally do self.request.get() or similar in other handlers? I've never used self.request.params...

I don't mind it in principle but if we do it here should we also do it in all other handlers for consistency? Alternatively it might be simpler to just use what the other handlers use.